.circleci/config/jobs/@docker.yml

```diff
@@ -6,6 +6,9 @@ docker-image:
       default: false
     tag:
       type: string
+    src:
```

Comment:
Not sure if you tested this and it works by indirect passing the parameter down to the all job steps, but could you please pass it explicitly to the `docker_image` and then to do `docker_tag` command